# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libprelude/fix-ltdl.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- libprelude-1.0.0.orig/src/include/prelude-plugin.h	2010-11-06 21:30:18.000000000 +0300
+++ libprelude-1.0.0/src/include/prelude-plugin.h	2010-11-06 21:30:47.000000000 +0300
@@ -61,9 +61,8 @@
  */
 #ifdef PRELUDE_APPLICATION_USE_LIBTOOL2
 # define lt_preloaded_symbols lt__PROGRAM__LTX_preloaded_symbols
-#endif
-
 extern const void *lt_preloaded_symbols[];
+#endif
 
 #define PRELUDE_PLUGIN_SET_PRELOADED_SYMBOLS()         \
         prelude_plugin_set_preloaded_symbols(lt_preloaded_symbols)

